src/MSBuild/LiveLogger/TerminalBuffer.cs

```diff
@@ -116,7 +117,10 @@ public static void Render()
                 overallBuildState == OverallBuildState.Warning ? ANSIBuilder.Formatting.ForegroundColor.Yellow :
                 ANSIBuilder.Formatting.ForegroundColor.White;
 
-            string text = $"MSBuild - Build in progress - {FinishedProjects} finished projects";
+            string text = IsRestoring ?
+                $"MSBuild - Restore in progress - {FinishedProjects} finished projects" :
+                $"MSBuild - Build in progress - {FinishedProjects} finished projects";
```

Comment:
Future item: I'd like the entrypoint project listed here, so the text becomes something like `Restoring foo.csproj` or `Building bar.sln`.